Source/GameBaseFramework/Private/GameFramework/Experiences/GBFExperienceDefinition.cpp

```diff
@@ -15,6 +18,78 @@ FPrimaryAssetType UGBFExperienceDefinition::GetPrimaryAssetType()
     return PrimaryAssetType;
 }
 
+TArray< FString > UGBFExperienceDefinition::GetAllGameFeatures( const UWorld * world ) const
+{
+    auto result = GameFeaturesToEnable;

```

Comment:
This creates a copy. Turns out it was better to fill the arrays by reference.
In addition to doing that:
* Use AddRange to add the whole `GameFeaturesToEnable` at once
* Reserve enough items in the array before passing it to this function (like twice the number of game features)

Do the same for actions

Reply:
Done.